File: examples/java/src/main/java/SimpleQRCode.java
import qrcode.QRCode;

import java.io.FileOutputStream;
import java.io.IOException;

public class SimpleQRCode {

    public void createQRCode(String content) throws IOException {
        FileOutputStream fileOut = new FileOutputStream("java-simple.png");
        new QRCode(content).render().writeImage(fileOut);
    }

    public static void main(String[] args) throws Exception {
        new SimpleQRCode().createQRCode("Hello, world!");
    }
}

File: examples/kotlin/src/main/kotlin/SimpleQRCode.kt
import qrcode.raw.QRCodeBuilder
import java.io.FileOutputStream

class SimpleQRCode {
    fun createQRCode(content: String) {
        FileOutputStream("kotlin-simple.png").use {
            QRCodeBuilder(content).render().writeImage(it)
        }
    }
}

fun main() {
    SimpleQRCode()
        .createQRCode("Hello, world!")
}
